tests/unittest/_torch/executor/test_py_scheduler.py

```
@@ -272,7 +283,7 @@ def test_max_num_tokens_limits_context(self):
             make_context_request(0, prompt_len=10),
             make_context_request(1, prompt_len=10),
         ]
-        ctx, gen = scheduler.schedule(requests, set())
+        _enc, ctx, gen = scheduler.schedule(requests, set())
```

Ruff (0.15.15)

[warning] Unpacked variable `gen` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)